src/js/components/Responsive/doc.js

```diff
@@ -0,0 +1,13 @@
+import { schema, PropTypes } from 'react-desc';
+
+export default Responsive => schema(Responsive, {
+  description: 'A react component that handles responsive breakpoints.',
+  usage: `import { Responsive } from 'grommet';
+  <Responsive onNarrow={() => {}} />`,
+  props: {
+    onNarrow: [
+      PropTypes.func,
+      'Function that will be called when the browser window crosses the narrow width.',
```

Comment:
I believe we cannot call this function `onNarrow`. I immediately thought that that we would have something like `onLapAndUp`. where onNarrow actually represents the change in the breakpoint. How about naming it `onChange` ?

Reply:
Changed to `onChange`.